src/SingleView/SingleView.js

```diff
@@ -5,7 +5,22 @@ import './SingleView.css'
 export class SingleView extends Component {
   constructor(props) {
     super(props)
-    this.state = {}
+    this.state = { currentMovie: { id: props.currentMovieID.id } }
+  }
+
+  fetchData = (path) => {
```

Comment:
I like this component because I think if you go directly to the `/movies/5` URL, as an example,  the page would still work. Try out this scenario to be sure. Imagine you're sending your friend a link to a specific movie page.

Reply:
This does work, we will make sure to add a test to verufiy!